package.json

```diff
@@ -22,7 +24,7 @@
     "url": "https://github.com/nf-core/nf-co.re/issues"
   },
   "homepage": "https://github.com/nf-core/nf-co.re#readme",
-  "devDependencies": {
+  "dependencies": {
```

Comment:
It's pretty standard. I don't know why the runtime requirements ended up in devDependencies in the first place, but I'm also questioning everything I know since it seemed to work fine with them in devDependencies, so `npm` has just been lying to me this whole time. :thinking:

Reply:
I might have always installed them as devdep, because in my point of view non of them are really runtime requirements (except for maybe sass) 😬

Reply:
Okay, I don't understand enough of what's happening under the hood, so maybe start a PR splitting them up?